app/src/main/java/com/example/concordia_campus_guide/Activities/RoutesActivity.java

```diff
@@ -96,6 +126,17 @@ private void openSearchPage(){
         startActivity(openSearch);
     }
 
+    private void setFrom() {
+        Coordinates toCoordinates = (mViewModel.getTo().getCenterCoordinates());
+        to = new MarkerOptions().position(new LatLng(toCoordinates.getLatitude(), toCoordinates.getLongitude()));
+
+    }
+
+    private void setTo() {
+        Coordinates fromCoordinates = (mViewModel.getFrom().getCenterCoordinates());
+        from = new MarkerOptions().position(new LatLng(fromCoordinates.getLatitude(), fromCoordinates.getLongitude()));
+    }
+
```

Comment:
Maybe I don't understand this right.. but why in the setFrom() you are setting the to, and vice versa for the setTo() ?

Reply:
oh oups I made a mistake here, thanks for catching it :)